duckduckgo/Android
# Update ktlint to 0.39

Task/Issue URL: https://app.asana.com/0/414730916066338/1199145345336726/f
Tech Design URL: 
CC: 

**Description**:
We updated kotlin to 1.4 version however ktlint is still in 0.36, which does not support the new features in kotlin 1.4. This means that if we try to use any of the new features in 1.4, ktlint will complain.

(Support for 1.4 was added in ktlint in 0.38)


**Steps to test this PR**:
1. fresh install the app
2. check core funcionality works

1. update an already install app
2. check core functionality works 


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
